src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.data;
+
+import java.util.HashSet;
+
+public final class User {
+  private final String email;
+  private final HashSet<Long> groups;
```

Comment:
it looks like in other parts of the application we're using a string for the groupId. are you planning on parsing it out into a long here?

Reply:
nvm I just saw that you're using Long.valueof() :)